kickstarts/partials/post/ui_compile.ks.erb

```diff
@@ -1,4 +1,4 @@
-npm install -g gulp bower
+npm install -g gulp bower yarn
 
 pushd /var/www/miq/vmdb
   bower -F --allow-root install
```

Comment:
The references to bower will be removed from bin/setup and bin/update?  cc @himdel